modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -0,0 +1,81 @@
+[[contact-methods-salt]]
+= Salt Bundle
+
+== What is Salt Bundle?
+
+The Salt Bundle is a single binary package containing Salt Minion, Python 3, required Python modules and libraries.
+
+As the Salt Bundle is shipped with Python 3 and all the requirements for Salt to run, the Salt Bundle is not using the Python version coming from the system, so it can be installed on clients which doesn't meet the requirements for a given Salt version.
+
+It is also possible to install Salt Bundle on the system where Salt Minion is being used already to manage that client by the different Salt Master than the {productname} Server.
```

Comment:
```suggestion
It is also possible to use the Salt Bundle on systems that run a Salt Minion connected to a non-{poductname} Salt Master.
```

Reply:
Not really sure if `non-Uyuni` or `non-SUSE Manager` cold look nice there.
Maybe do it this way:
```suggestion
It is also possible to use the Salt Bundle on systems that run a Salt Minion connected to a Salt Master other than {poductname} Salt Master.
```

Reply:
You're right, `non-SUSE Manager` looks bad. I tried to avoid repeating "Salt Master", but I agree that your version is better than mine.